src/utils/tar.zig

```diff
@@ -87,8 +88,10 @@ pub fn parallelUntarToFileSystem(
     var file_count: usize = 0;
     const strip_components: u32 = 0;
     loop: while (true) {
-        var header_buf = try allocator.alloc(u8, 512);
-        _ = try reader.readAtLeast(header_buf, 512);
+        const header_buf = try allocator.alloc(u8, 512);
+        if (header_buf.len != try reader.readAtLeast(header_buf, 512)) {
```

Comment:
can we change `header_buf.len` to hardcoded `512` in this `==` check -- just easier to read